spec/features/paginate_posts_index_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+describe 'Paginate posts index' do
+  subject { page }
+  describe 'index is paginated' do
+    before do
+      25.times { FactoryGirl.create(:blog_post) }
```

Comment:
Usually you want to avoid "magic numbers" like the 25 here. It doesn't tell you why that number is there or what does it represent. So what you want to do is give it a name like so:

```
POSTS_PER_PAGE = 25
POSTS_PER_PAGE.times { ... something ... }
```

Also it's preferable to use the same constant you use in the implementation code. For example you can define a constant in the `BlogPost` model like `BlogPost::POSTS_PER_PAGE` and you can use that everywhere (both implementation and spec).